docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SandboxSpineClient.java

```diff
@@ -20,13 +26,18 @@ public Response sendAcsData(String requestBody) {
 
     @Override
     public Response sendScrData(String requestBody, String nhsdAsid, String nhsdIdentity, String nhsdSessionUrid) {
-        return null;
+        Header[] headers = {
+            new BasicHeader(CONTENT_LOCATION_HEADER, ""),
+            new BasicHeader(RETRY_AFTER_HEADER, "1000")
```

Comment:
1000 means that app will start polling after 1000ms. Maybe we could use a smaller value to improve sandbox users experience? Maybe 300?

Reply:
Time reduced to 300ms.